generators/php/codegen/src/asIs/Constant.Template.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace <%= namespace%>;
+
+use DateTimeInterface;
+
+class Constant
+{
+    public const DateFormat = 'Y-m-d';
+    public const DeserializationDateFormat = "!" . self::DateFormat;
```

Comment:
nit: Thoughts on this instead?

```suggestion
    public const DateDeserializationFormat = "!" . self::DateFormat;
```

Reply:
done ✅